src/bsp/src/stm32/src/InterlocManager.cpp

```diff
@@ -1,7 +1,26 @@
 #include "InterlocManager.h"
 #include "bsp/BSPContainer.h"
 #include <Task.h>
+#define NB_CALIB_TRIES 10
 
+bool isFrameGood(UWBRxFrame frame, ILogger& logger) {
+
+    if (frame.m_status == UWBRxStatus::FINISHED) {
+        // logger.log(LogLevel::Warn,"Good message");
+    } else if (frame.m_status == UWBRxStatus::ERROR) {
+        logger.log(LogLevel::Warn, "Message in error");
```

Comment:
be more verbose in the logging 

"InterlocManager: UWB error message"

same for the other logging under it